src/AWS.Deploy.Orchestration/CdkProjectHandler.cs

```diff
@@ -69,6 +73,21 @@ public async Task<string> ConfigureCdkProject(OrchestratorSession session, Cloud
             return cdkProjectPath;
         }
 
+        public async Task<string> PerformCdkDiff(string cdkProjectPath, CloudApplication cloudApplication)
```

Comment:
Throughout this entire call stack - can you document what the return type is? Based on the function name assumed it'd be the `cdk diff` output, but looks like it's the full body of the CloudFormation template instead?

Reply:
I added documentation on the call stack